pkg/api/filter/auth/keystone.go

```diff
@@ -84,27 +83,28 @@ func (k *Keystone) setPolicyContext(r tokens.GetResult) error {
 
 	project, err := r.ExtractProject()
 	if err != nil {
-		return model.HttpError(k.ctx, http.StatusUnauthorized, "extract project failed,%v", err)
+		return model.HttpError(ctx, http.StatusUnauthorized, "extract project failed,%v", err)
 	}
 
 	user, err := r.ExtractUser()
 	if err != nil {
-		return model.HttpError(k.ctx, http.StatusUnauthorized, "extract user failed,%v", err)
+		return model.HttpError(ctx, http.StatusUnauthorized, "extract user failed,%v", err)
 	}
 
-	ctx := &context.Context{
-		TenantId:       project.ID,
-		Roles:          roleNames,
-		UserId:         user.ID,
-		IsAdminProject: project.Name == "admin",
+	param := map[string]interface{}{
+		"TenantId":       project.ID,
+		"Roles":          roleNames,
+		"UserId":         user.ID,
+		"IsAdminProject": project.Name == "admin",
```

Comment:
This check should be case in-sensitive.  "admin" could be "Admin", etc.

Reply:
@xing-yang You mean `admin` is equal to `Admin`?

Reply:
I meant it is possible the admin user is named "Admin" instead of "admin".  Both should be fine.  By the way, is there a better way of checking this rather than a string comparison?  Is there a method we can call?

Reply:
using strings.Tolower may be better.